components/login/login_controller/login_controller.jsx

```diff
@@ -733,6 +741,37 @@ class LoginController extends React.PureComponent {
             );
         }
 
+        if (openIdSigninEnabled) {
+            const buttonStyle = {};
+            if (this.props.openidButtonColor) {
+                buttonStyle.backgroundColor = this.props.openidButtonColor;
+            }
+            var buttonText = (
+                <FormattedMessage
+                    id='login.openid'
+                    defaultMessage='Open Id'
+                />);
```

Comment:
I'd say maybe stylistically we'd want the closing bracket to align with our variable. 
```suggestion
                />
            );
```